social-network/src/CardMissedIt.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import bookmark from './noun_bookmark_48824.png';
+import audio from './noun_Audio_830722.png';
+import star from './noun_Star_7371.png';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Container from 'react-bootstrap/Container';
+
+export default class CardMissedIt extends React.Component {
```

Comment:
Note that a lot of this component is duplicated from the other article component, could be a good opportunity to merge in order to reduce the amount of code.